src/styles/divider.css

```diff
@@ -0,0 +1,13 @@
+/* divider */
+
+.ui.divider {
+  background-color: var(--neutral-500);
+  margin: 0;
+  flex-shrink: 0;
+  width: 100%;
+  height: 1px;
```

Comment:
Let's add `text-transform: none;` here, in the case `horizontal` or `vertical` are being used.